code-challenges.test.js

```diff
@@ -1,30 +1,103 @@
+// // the starting values for index 0 and 1 are 1 and 1
+// // [1,1]
+// // for index 2 you add index 0 and 1
+// // place value inside array
+// // [1,1,2]
+// // for index 3 you add index 1 and 2
+// // place value inside array
+// // [1, 1, 2, 3]
+// // place value inside array and keep going with rest of the numbers
+// // return array in same length with fibonacci
+
+// // a) Create a test with expect statements for each of the variables provided.
+
+describe("greaterThanTwo", () => {
+  it("takes in a number (greater than 2) and returns an array, that length, containing the numbers of the Fibonacci sequence", () => {
+    const fibLength1 = 6;
+// //     // Expected output: [1, 1, 2, 3, 5, 8]
+
+    const fibLength2 = 10;
+// //     // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+
+    expect(greaterThanTwo(fibLength1)).toEqual([1, 1, 2, 3, 5, 8]);
+    expect(greaterThanTwo(fibLength2)).toEqual([
+      1, 1, 2, 3, 5, 8, 13, 21, 34, 55,
+    ]);
+  });
+});
+
+// // initial test output:
+// // --->  FAIL  ./code-challenges.test.js
+// //  ReferenceError: greaterThanTwo is not defined
+
+// // psuedocode:
+// // input: 6
+// // output: [1, 1, 2, 3, 5, 8]
+// // fibonacci: these are your starting values of fibonacci [1, 1, to get the next number add the two numbers right before 1+1=2 => 1, 1, 2 then 2+1=3 => 1, 1, 2, 3]
+// // the starting values for index 0 and 1 are 1 and 1
+// // [1,1]
+// // for index 2 you add index 0 and 1
+// // place value inside array
+// // [1,1,2]
+// // for index 3 you add index 1 and 2
+// // place value inside array
+// // [1, 1, 2, 3]
+// // place value inside array and keep going with rest of the numbers
+// // return array in same length with fibonacci
+
+// // b) Create the function that makes the test pass.
+
+const fibLength1 = 6;
+// // Expected output: [1, 1, 2, 3, 5, 8]
+
+const fibLength2 = 10;
+// // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+
+const fibonacciArray = [1, 1];
+for (let i = 0; i < 10; i++) {
+  fibonacciArray[0] = fibonacciArray[2 - 2];
+  fibonacciArray[1] = fibonacciArray[2 - 1];
+  console.log(fibonacciArray(fibLength1));
+}
+const greaterThanTwo = (number) => {
 
+  return newArray.push(fibonacciArray[i - 2] + fibonacciArray[i - 1])
+  })
+}
+// console/log(greaterThanTwo)
 // --------------------2) Create a function that takes in an object and returns an array of the values sorted from least to greatest.
 // Hint: Check out this resource: Object.values() https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values
 
+// psuedocode: Create a function named studyMinutes that takes in an object and returns an array of the values sorted from least to greatest.
+// Input: Object
+// Output: Array with values sorted least to greatest
+// create function called studyMinutes that takes in and object and outputs an array
+// I'm thinking I can iterate over the array using the sort method
```

Comment:
Good attempt. So when doing the pseudocode, remember to approach the datatype. You are starting with an object. What information do you need to grab from the object? Once you grab that data, how do you turn it into an array? Sort is definitely a good method to try after you gather your data.